src/main/java/util/ErrorMessage.java

```diff
@@ -0,0 +1,21 @@
+package util;
+
+public enum ErrorMessage {
+    ERROR_NAME_LENGTH("자동차 이름은 한 글자 이상 다섯 글자 이하여야 합니다."),
+    ERROR_IS_BLANK_OR_NULL("자동차 이름을 입력해 주십시오."),
+    ERROR_DUPLICATED_NAME("자동차의 이름은 중복될 수 없습니다."),
+    ERROR_NAME_COUNT("자동차 이름은 2개 이상이어야 합니다."),
+    ERROR_ROUND_IS_NULL("시도 횟수를 입력해 주십시오."),
+    ERROR_ROUND_IS_NATURAL_NUMBER("시도 횟수는 자연수여야 합니다.");
+
+    private static final String PREFIX_ERROR = "[ERROR] ";
+    private String message;
```

Comment:
```suggestion
    private static final String PREFIX_ERROR = "[ERROR] ";
    
    private String message;
```
엄격하게 준수해야하는 컨벤션은 아니지만 이렇게 상수와 변수 사이에 공백을 추가하면 더 잘 읽혀요!